examples/concept.html

```diff
@@ -10,7 +10,7 @@
     <link rel="stylesheet" href="../css/ol3gm.css" type="text/css" />
     <link rel="stylesheet" href="./resources/layout.css" type="text/css" />
   </head>
-  <body>
+  <body>i
```

Comment:
@adube This was a mistake, likely way jumping into iTerm, Cmd+Space…i… If you can remove this or I can do another PR.

Reply:
I'll remove it.

Reply:
I'm also about to make a PR to fix an issue with `olgm.herald.Feature` not syncing styles, it honors styles if they're defined when a feature is added to a source, but it is not listening for the change to styles or to the `propertychange` event so it never updates the styles. This is breaking my use, where I only set styles to points after I have the image loaded, that was I can deal with the image failing to load and leave it as a default circle for the point.

Reply:
Indeed, it does.  Okay, I'll wait.

Reply:
Thanks @adube